seshdash/models.py

```diff
@@ -89,6 +89,42 @@ def __str__(self):
         return self.name
 
 
+class Status_Card(models.Model):
+     """ 
+     Contains the rows to be displayed in the 
+     status card
+     """
+   
+     ROW_CHOICES = (
+         ('AC_Load_in', 'AC Load in'),
+         ('AC_Load_out', 'AC Load out'),
+         ('AC_Voltage_in', 'AC Voltage in'),
+         ('AC_Voltage_out', 'AC Voltage out'),
+         ('AC_input', 'AC input'),
+         ('AC_output', 'AC output' ),
+         ('AC_output_absolute', 'AC output absolute'),
+         ('battery_voltage', 'Battery Voltage'),
+         ('genset_state', 'Genset state'),
+         ('main_on', 'Main on'),
+         ('pv_production', 'PV production'),
+         ('relay_state', 'Relay state'),
+         ('soc', 'State of Charge'),
+         ('trans', 'Trans'),
+     )
+
+
+     row1 = models.CharField(max_length=30, choices=ROW_CHOICES)
```

Comment:
can you add the  defaults for the rows?
The defaults can be what they are set to now.